net/grpc.go

```diff
@@ -63,6 +75,30 @@ type serviceServer interface {
 	PushLog(context.Context, *pushLogRequest) (*pushLogReply, error)
 	// GetHeadLog from this peer
 	GetHeadLog(context.Context, *getHeadLogRequest) (*getHeadLogReply, error)
+	GetIdentity(context.Context, *getIdentityRequest) (*getIdentityReply, error)
+}
+
+func getIdentityHandler(
+	srv any,
+	ctx context.Context,
+	dec func(any) error,
+	interceptor grpc.UnaryServerInterceptor,
+) (any, error) {
+	in := new(getIdentityRequest)
+	if err := dec(in); err != nil {
+		return nil, err
+	}
+	if interceptor == nil {
+		return srv.(serviceServer).GetIdentity(ctx, in)
+	}
+	info := &grpc.UnaryServerInfo{
+		Server:     srv,
+		FullMethod: serviceGetIdentityName,
+	}
+	handler := func(ctx context.Context, req any) (any, error) {
+		return srv.(serviceServer).GetIdentity(ctx, req.(*getIdentityRequest))
+	}
+	return interceptor(ctx, in, info, handler)
```

Comment:
Reaching this line depends on the GRPC server behaviour. Since it's reused from generated code, I'm not interested in creating mocks just to get full coverage.

Reply:
I didn't know it's reused from generated code. thanks for explaination.